camunda-modeler-plugin/main/bpmndt/PathFinder.js

```diff
@@ -196,8 +196,14 @@ export default class PathFinder {
           return this._getEscalationBoundary($parent.id, eventDefinitions[0].escalationRef?.escalationCode);
         } else {
           // handle end events of embedded sub processes
-          const incoming = this._findIncoming($parent.id);
-          return incoming ? [ incoming.id ] : [];
+          this._findIncoming($parent.id).map(element => {
```

Comment:
do we really expect multiple outgoing sequence flows from a subprocess - normally this is a modeling error